src/main/routing/resolvers/useUrlParser.ts

```
@@ -0,0 +1,108 @@
+        const parsedUrl = new URL(url);
+        if (parsedUrl.origin === window.location.origin) {
+          for (const parser of parsers) {
+            const match = matchPath(
+              {
+                path: parser,
+                caseSensitive: true,
+                end: true,
+              },
+              parsedUrl.pathname
+            );
+            if (match && match.params) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)